app/src/main/res/values/strings.xml

```diff
@@ -138,6 +138,17 @@
   <string name="add_profile_error_name_only_letters">Names can only have letters. Try another name?</string>
   <string name="add_profile_error_pin_length">Your PIN should be 3 digits long.</string>
   <string name="add_profile_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
+  <!-- OnboardingActivity -->
+  <string name="slide_0_title">Welcome to Oppia!</string>
```

Comment:
Here and below: suggest including ‘onboarding’ in the names to make their context clearer.

Reply:
Done here and everywhere else.